mlrun/runtimes/sparkjob.py

```diff
@@ -159,6 +162,10 @@ def _run(self, runobj: RunObject, execution: MLClientCtx):
         update_in(job, "spec.executor.instances", self.spec.replicas or 1)
         if self.spec.image:
             update_in(job, "spec.image", self.spec.image)
+        elif environ.get("IGZ_VERSION"):
+            update_in(
+                job, "spec.image", "iguazio/spark-app:{}".format(environ["IGZ_VERSION"])
```

Comment:
Please add this to config.py in line 51 (this will automatically read from the env vars `MLRUN_IGZ_VERSION` and `MLRUN_SPARK_IMAGE`):
```
"igz_version": "",  # the version of the iguazio system the API is running on
"spark_image": "iguazio/spark-app",  # image to use for spark runtime
```

Then in your code here read from config:
```
from mlrun.config import config
config.igz_version
config.spark_image
```
and please add logic so that you're using the `igz_version` as the image tag only if the `spark_image` don't have tag already (search for `:`), that way we can override it from outside